feat(apps-ui-kit): add iota names color variables to UI-KIT components

Juligs opened the pull request:
fixes #7406

Juligs committed:
refactor(apps-ui-kit): modify ui-kit config to allow multitheme (#7309)

fixes #7300

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
refactor(apps-ui-kit): modify ui-kit config to allow multitheme (#7309)

fixes #7300

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
feat(apps-ui-kit): add iota-names color palette and typography (#7331)

fixes #7302

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
feat(apps-ui-kit): add iota-names color palette and typography (#7331)

fixes #7302

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
feat(apps-ui-kit): add iota-names color palette and typography (#7331)

fixes #7302

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
refactor(apps-ui-kit):  update atoms to apply new palettes (Badge-LabelText) (#7355)

fixes #7344

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
refactor(apps-ui-kit):  update atoms to apply new palettes (Badge-LabelText) (#7355)

fixes #7344

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

Juligs committed:
refactor(apps-ui-kit):  update atoms to apply new palettes (Badge-LabelText) (#7355)

fixes #7344

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

evavirseda committed:
feat(apps-ui-kit): update atoms to apply new palettes (`LisItem` - `VisualAssetCard`) (#7349)

# Description of change

Please write a summary of your changes and why you made them.

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7345

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update atoms to apply new palettes (`LisItem` - `VisualAssetCard`) (#7349)

# Description of change

Please write a summary of your changes and why you made them.

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7345

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update molecules to apply new palettes (#7354)

# Description of change

nested PR :warning: https://github.com/iotaledger/iota/pull/7349

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7351

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update molecules to apply new palettes (#7354)

# Description of change

nested PR :warning: https://github.com/iotaledger/iota/pull/7349

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7351

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update molecules to apply new palettes (#7354)

# Description of change

nested PR :warning: https://github.com/iotaledger/iota/pull/7349

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7351

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update organisms to apply new palettes (#7359)

# Description of change

nested PR :warning: https://github.com/iotaledger/iota/pull/7354

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7352

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update organisms to apply new palettes (#7359)

# Description of change

nested PR :warning: https://github.com/iotaledger/iota/pull/7354

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7352

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
feat(apps-ui-kit): update organisms to apply new palettes (#7359)

# Description of change

nested PR :warning: https://github.com/iotaledger/iota/pull/7354

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7352

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
bricg back border rounded

evavirseda committed:
fix neutral badge border

evavirseda committed:
fix class

evavirseda committed:
fix keyvalue icon color

evavirseda committed:
fix class

evavirseda committed:
fix outline border

evavirseda committed:
filter themes to not show names

evavirseda committed:
fix class

evavirseda committed:
fix tooltip in the wallet

Juligs committed:
feat(apps-ui-kit): update storybook after multi-theme (#7377)

fixes #7374

Juligs committed:
feat(apps-ui-kit): update storybook after multi-theme (#7377)

fixes #7374

Juligs committed:
feat(apps-ui-kit): update storybook after multi-theme (#7377)

fixes #7374

evavirseda committed:
Merge branch 'tooling/make-ui-kit-multi-theme-compatible' into tooling-uikit/fix-missing-styles

evavirseda committed:
fix fonts

Juligs committed:
fix(apps-ui-kit): adjust styles for disabled state and improve label

Juligs committed:
feat(apps-ui-kit): add some color variables to componets

evavirseda committed:
fix navbar item classes

Juligs committed:
refactor(apps-ui-kit): fix search component styles, add divider color variables, and update Storybook backgrounds

Juligs committed:
Merge branch 'tooling-uikit/fix-missing-styles' into feat/add-iota-names-color-variables

Juligs committed:
feat(apps-ui-kit): add some color names variables

Juligs committed:
feat(apps-ui-kit): update search and table cell styles with new color variables

Juligs committed:
Merge branch 'tooling-uikit/fix-missing-styles' into feat/add-iota-names-color-variables

Juligs committed:
fix(apps-ui-kit): update table header cell, title, and accordion styles

Juligs committed:
feat(apps-ui-kit): update table header cell, title, and accordion variables

Juligs committed:
Merge branch 'tooling-uikit/fix-missing-styles' into feat/add-iota-names-color-variables

Juligs committed:
feat(apps-ui-kit): add color variables for buttons

Juligs committed:
fix(apps-ui-kit): correct input border color classes and improve focus/error state handling

Juligs committed:
chore(apps-ui-kit): fix prettier format

Juligs committed:
Merge branch 'tooling-uikit/fix-missing-styles' into feat/add-iota-names-color-variables

Juligs committed:
Revert "feat(apps-ui-kit): update table header cell, title, and accordion variables"

This reverts commit ae15f0b2e9704d2a244a629dc0763f3e9a8be212.

Juligs committed:
feat(apps-ui-kit): update styles for table header cell, title icon, and accordion border

Juligs committed:
Merge branch 'tooling-uikit/fix-missing-styles' into feat/add-iota-names-color-variables

Juligs committed:
feat(apps-ui-kit): update color variables for various UI components

evavirseda committed:
Merge branch 'tooling/make-ui-kit-multi-theme-compatible' into tooling-uikit/fix-missing-styles

evavirseda committed:
fix missing conflict

evavirseda commented on apps/ui-kit/src/lib/styles/index.css:
```diff
@@ -612,16 +608,16 @@
     /* TOOLTIP */
-    --tooltip-bg: theme('colors.names-neutral.30');
-    --tooltip-text-color: theme('colors.names-neutral.92');
+    --tooltip-bg: theme('colors.black');
+    --tooltip-text-color: theme('colors.white');
```
```suggestion
    --tooltip-bg: theme('colors.names-neutral.0');
    --tooltip-text-color: theme('colors.names-neutral.100');
```

evavirseda reviewed (changes requested):
just one detail to keep consistence :pray:

evavirseda committed:
Merge branch 'tooling-uikit/fix-missing-styles' into feat/add-iota-names-color-variables

Juligs committed:
chore(apps-ui-kit): update color variables

evavirseda committed:
feat(apps-ui-kit): review apps and fix missing styles (#7376)

# Description of change

Please write a summary of your changes and why you made them.

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/7375

## How the change has been tested

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

### Release Notes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

---------

Co-authored-by: Juliana <115430927+Juligs@users.noreply.github.com>
Co-authored-by: Juligs <julicodes29@gmail.com>

evavirseda committed:
refactor(apps-ui-kit): modify ui-kit config to allow multitheme (#7309)

fixes #7300

---------

Co-authored-by: evavirseda <evirseda@boxfish.studio>

evavirseda committed:
feat(apps-ui-kit): update atoms to apply new palettes (`LisItem` - `VisualAssetCard`) (#7349)

Please write a summary of your changes and why you made them.

fixes https://github.com/iotaledger/iota/issues/7345

- [ ] Basic tests (linting, compilation, formatting, unit/integration
tests)
- [ ] Patch-specific tests (correctness, functionality coverage)
- [ ] I have added tests that prove my fix is effective or that my
feature works
- [ ] I have checked that new and existing unit tests pass locally with
my changes

- [ ] Protocol:
- [ ] Nodes (Validators and Full nodes):
- [ ] Indexer:
- [ ] JSON-RPC:
- [ ] GraphQL:
- [ ] CLI:
- [ ] Rust SDK:
- [ ] REST API:

evavirseda committed:
Merge branch 'tooling/make-ui-kit-multi-theme-compatible' into feat/add-iota-names-color-variables

evavirseda committed:
fix fmt

VmMad reviewed (changes requested):
1. Badge neutral has a border that doesn't exist in figma ![image](https://github.com/user-attachments/assets/1b006b70-709d-4d8d-b838-0580d9aef017)
2. ButtonSegment type underlined should have border-radius in the highlighted border (left deploy, right figma)
![image](https://github.com/user-attachments/assets/bcb14d50-0b23-49a6-afeb-877776686855)
3. In KeyValueInfo, the "label" text has a different color. ALso the supporting text has a slightly different color `#6E7787` → `#71748B`
![image](https://github.com/user-attachments/assets/9957ffdf-2e47-4653-ad72-b299a7a8b58f)
![image](https://github.com/user-attachments/assets/fe4f68e4-d715-4270-be21-deccd72a5cfc)
4. Title; subtitle has a different text color, also the tooltip icon has different color 
![image](https://github.com/user-attachments/assets/38ccffba-1510-4358-a12a-936d494d97c3)
5. VisualAssetCard has different color on hover 
![image](https://github.com/user-attachments/assets/14a19a64-de45-4c8c-b187-804d0e89aadf)
6. Scrollbar has different color 
![image](https://github.com/user-attachments/assets/9e1b273f-b299-4e90-ac69-e0f23164f915)
7. Chip; Text inside has different color 
![image](https://github.com/user-attachments/assets/aa076b38-95fd-4353-8961-b44a89241b9f)
8. In the dialog i noticed 3 things:
    a. The background color is not correct. It should be `#0F1228`, current is `#ACAFC5`
    b. The overlay should be `#000000 / 72`, current is set to `bg-shader-neutral-light-48`.
    c. The figma dialog has an outline.

VmMad committed:
fix: styles for some commponents

VmMad committed:
Merge branch 'tooling/make-ui-kit-multi-theme-compatible' into feat/add-iota-names-color-variables

Juligs committed:
Merge branch 'tooling/make-ui-kit-multi-theme-compatible' into feat/add-iota-names-color-variables
